tests/storage-disks-vm

```diff
@@ -322,24 +332,37 @@ if lxc exec v1 -- mount | grep -qwF /boot/efi; then
 fi
 
 if hasNeededAPIExtension disk_io_bus; then
-  # Remove disk device restrictions and add a NVMe disk
+  # Remove disk device restrictions and add disks
   lxc stop -f v1
   lxc config device remove v1 d1
   lxc project unset restricted restricted.devices.disk
   lxc project unset restricted restricted.devices.disk.paths
   lxc project unset restricted restricted
 
-  # Add a NVMe disk and check if a NVMe controller is added to the VM
-  lxc config device add v1 nvme-ssd disk source="${loopdev}" io.bus=nvme
+  # Add a NVMe disk
+  lxc config device add v1 nvme-ssd disk source="${loopdev1}" io.bus=nvme
+  
+  # Add a virtio-blk disk
+  lxc config device add v1 virtio-blk-ssd disk source="${loopdev2}" io.bus=virtio-blk
+
+  # Add a virtio-scsi disk
+  lxc config device add v1 virtio-scsi-ssd disk source="${loopdev3}" io.bus=virtio-scsi
+
+  # Start VM
   lxc start v1
-  waitInstanceReady v1
+  waitInstanceReady v1 
+
+  # Check if NVMe, virtio-blk and virtio-scsi controllers are added to the VM
   if ! lxc exec v1 -- sh -c 'command -v lspci'; then
     lxc exec v1 -- apt-get update
     lxc exec v1 -- apt-get install --no-install-recommends -y pciutils
   fi
```

Comment:
For debuggability, we should add a `lxc exec v1 -- lspci` right here. This way, we'd better understand why the `grep`s are not matching.